app/Http/Controllers/Auth/VATSIMOAuthController.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Http\Controllers\Auth;
+
+use League\OAuth2\Client\Provider\Exception\IdentityProviderException;
+use League\OAuth2\Client\Provider\GenericProvider;
+
+class VATSIMOAuthController extends GenericProvider
+{
+    /**
+     * @var GenericProvider
+     */
+    private $provider;
+
+    /**
+     * Initializes the provider variable.
+     */
+    public function __construct()
+    {
+        parent::__construct([
+            'clientId'                => config('vatsim-connect.id'),
+            'clientSecret'            => config('vatsim-connect.secret'),
+            'redirectUri'             => route('login.post'),
+            'urlAuthorize'            => config('vatsim-connect.base').'/oauth/authorize',
```

Comment:
```suggestion
            'urlAuthorize'            => "{config('vatsim-connect.base')}'/oauth/authorize",
```
See if that works? Looks a little bit cleaner.

Reply:
Not sure this is much cleaner, but it didn't work anyhow!

Reply:
might not even need the curly braces anymore, nae bother